guvnor-structure/guvnor-structure-backend/src/main/java/org/guvnor/structure/backend/config/ConfigurationFactoryImpl.java

```diff
@@ -30,6 +30,14 @@
     @Inject
```

Comment:
As you're moving to constructor injection please remove the [instance variable injection](https://github.com/bbrodt/guvnor/blob/18f93d3d0b26bb9d8771e6614ce583b01e33a1ae/guvnor-structure/guvnor-structure-backend/src/main/java/org/guvnor/structure/backend/config/ConfigurationFactoryImpl.java#L42).